webtv/src/components/GrantAuthDevice/GrantAuthDevice.tsx

```diff
@@ -0,0 +1,187 @@
+import { AUTH_SESSION_KEY } from "../../constants/auth.constants";
+import { UseOpenPassDeviceAuth } from "../../hooks/OpenPassDeviceAuth.hook";
+import {
+  Drawer,
+  DrawerBody,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  DrawerCloseButton,
+  useDisclosure,
+  Image,
+  Heading,
+  Box,
+  Divider,
+  Text,
+  Avatar,
+} from "@chakra-ui/react";
+import { useEffect, useMemo, useState } from "react";
+
+type GrantAuthDeviceProps = {
+  shouldOpen: boolean;
+  onClosed: () => void;
+};
+
+const SignIn = ({
+  deviceAuth,
+  errorState = false,
+}: {
+  errorState: boolean;
+  deviceAuth: { userCode?: string; verificationUriComplete?: string };
+}) => {
+  const getQRCodeURl = useMemo(() => {
+    const _userCode = deviceAuth?.userCode;
+    const _path = `https://myopenpass.com/code?user_code=${_userCode}`;
```

Comment:
You should be able to get this url from the result of the /authorize-device call. It's called `complete url` or something

Reply:
solved